src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -401,32 +406,6 @@ void ResumptionDataDB::GetDataForLoadResumeData(
   SelectDataForLoadResumeData(saved_data);
```

Comment:
@Kozoriz can you please check 
bool ResumptionDataDB::RemoveApplicationFromSaved.. a few lines above.